app/src/main/java/com/jetpack/compose/learning/instagramdemo/InstagramLoginActivity.kt

```diff
@@ -0,0 +1,184 @@
+package com.jetpack.compose.learning.instagramdemo
+
+import android.os.Bundle
+import androidx.compose.ui.graphics.Color
+import android.widget.Toast
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Visibility
+import androidx.compose.material.icons.filled.VisibilityOff
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.jetpack.compose.learning.R
+
+class InstagramLoginActivity: ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            Surface(modifier = Modifier.fillMaxSize()) {
+                LoginScreen()
+            }
+        }
+    }
+    
+    @Preview
```

Comment:
instead of adding @Prview here make 1 separate method and call this function from there. and on that separate method add @Preview